tests/data/valid/doc_string_escape/expected_default.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Docstrings
+
+  Scenario: Escaping docstrings
+    Given I have a docstring
+    """
+    This docstring has \"escaped\" double quotes.
+    It also has \`escaped\` backticks.
+    """
+    And I have a docstring with an escaped docstring in
+    """
+    This is another docstring.
+      \"\"\"
+      It has an escaped docstring inside it.
+      \"\"\"
+    """
+    And I have a backtick docstring
+    """
+    This docstring has \"escaped\" double quotes.
+    It also has \`escaped\` backticks.
+    """
+    And I have a nested single-quote docstring
+    """
+    This is a backtick docstring.
+      \`\`\`
```

Comment:
Ideally this would be `` ``` ``, not `` \`\`\` ``, but we can't do this as there appears to be a bug in the Gherkin parser.

Take the following two snippets:

This:

    ```
      \`\`\`
      Inner doc string
      \`\`\`
    ```

And this:

    """"
      \`\`\`
      Not an inner doc string
      \`\`\`
    """

The backticks appear as  `` \`\`\` `` in the Gherkin AST for both of these, so when we see `` \`\`\` `` in the AST we have no way of knowing what the original was. This seems to be a bug in Gherkin, so I'm going to file a bug report later on.

Reply:
Filed at cucumber/cucumber#865.

Reply:
Maybe we should turn this into an expected fail? Let me know what you think.

Reply:
Since this is an issue with the parser, I think we can leave it as is and watch for changes from the parser.

Reply:
OK, sounds good. If the Gherkin parser behaviour gets fixed then the above test should fail, so we should be aware of it when it happens.